app/javascript/utilities/color/accentCalculator.js

```diff
@@ -0,0 +1,58 @@
+
+
+/**
+ * Updates the brightness of the color
+ * @param {String} color
+ * @param {Integer} amount
+ * Based on the ruby implementation
+ * https://github.com/forem/forem/blob/main/app/services/color/compare_hex.rb
+ */
+export function brightness(color, amount = 1) {
```

Comment:
I added a utility class that calculates the brightness as discussed. The reason I put in a folder called `color` is because there will be more color utilities like validating the color contrast (which I'm working on in parallel)  so grouping them will be useful.